### PR TITLE
fix: Added hover tooltip for available envs []

### PR DESCRIPTION
**Pull request by Astitva877**
## Description
Added tooltip on hover for env variables 
image

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [x] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

File: src/lib/components/collections/req-res-section/sub-components/sub-components-header/CodeMirrorInput.svelte [hunk 1]
```diff
@@ -168,6 +168,37 @@
     );
   };
 
+  const wordHover = hoverTooltip((view, pos, side) => {
```
Comment (LordNayan) [inline src/lib/components/collections/req-res-section/sub-components/sub-components-header/CodeMirrorInput.svelte:171]: Have we repeated this code?

Comment (Astitva877) [reply src/lib/components/collections/req-res-section/sub-components/sub-components-header/CodeMirrorInput.svelte:171]: Yeah we have used hoverTooltip also but the use case was different like we were using it to check and color envs, I tried to add this use case in that code itself but that then it was not working as expected.

**Review by LordNayan — APPROVED** [review]

**Pull request merged**